### PR TITLE
py65: init at 1.1.0

### PR DESCRIPTION
**Pull request by AndersonTorres**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/misc/emulators/py65/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    description = "Emulate 6502-based microcomputer systems in Python";
+    longDescription = ''
+      Py65 includes a program called Py65Mon that functions as a machine
+      language monitor. This kind of program is sometimes also called a
+      debugger. Py65Mon provides a command line with many convenient commands
+      for interacting with the simulated 6502-based system.
+    '';
+    license = licenses.bsd3;
+    maintainers = with maintainers; [ AndersonTorres ];
+    platforms = with platforms; all;
```
Comment (SuperSandro2000) [inline pkgs/misc/emulators/py65/default.nix:24]: This shouldn't be set for buildPythonApplication.

Comment (AndersonTorres) [reply pkgs/misc/emulators/py65/default.nix:24]: Why?

Comment (SuperSandro2000) [reply pkgs/misc/emulators/py65/default.nix:24]: It is set by buildPythonApplication to the platforms python supports which are smaller than all.

Comment (AndersonTorres) [reply pkgs/misc/emulators/py65/default.nix:24]: Okay. I will fix this in the future.

**Pull request merged**